docs/codeql/codeql-language-guides/codeql-library-for-python.rst

```diff
@@ -20,8 +20,6 @@ The CodeQL library for Python incorporates a large number of classes. Each class
 
 -  **Syntactic** - classes that represent entities in the Python source code.
 -  **Control flow** - classes that represent entities from the control flow graphs.
--  **Type inference** - classes that represent the inferred values and types of entities in the Python source code.
--  **Taint tracking** - classes that represent the source, sinks and kinds of taint used to implement taint-tracking queries.
```

Comment:
I think it would be good to mention DataFlow nodes and API-graph nodes here as well (since they are 2 new categories). If this is planned for an other PR, just let me know :wink: 

Adding two more categories would also take the count back to 4, so we don't need to fix the `divided into four` text on line 19 :joy:

Reply:
Excellent point :-D I have added the two categories and given forward references (the reference to the API section commented out until #5389 is merged).

Reply:
Nice :+1: